openmetadata-ui/src/main/resources/ui/src/components/DataProducts/DataProductsContainer/DataProductsContainer.component.tsx

```diff
@@ -172,25 +176,66 @@ const DataProductsContainer = ({
     );
   }, [showHeader, dataProducts, hasPermission, showAddTagButton]);
 
+  const addTagButton = useMemo(
+    () =>
+      showAddTagButton ? (
+        <Col
+          className="m-t-xss"
+          data-testid="add-data-product"
+          onClick={handleAddClick}>
+          <TagsV1 startWith={TAG_START_WITH.PLUS} tag={TAG_CONSTANT} />
+        </Col>
+      ) : null,
+    [showAddTagButton]
+  );
+
+  const renderer = useMemo(() => {
+    if (isEditMode) {
+      return autoCompleteFormSelectContainer;
+    }
+
+    if (newLook && isEmpty(renderDataProducts)) {
+      return null;
+    }
+
+    return (
+      <Row data-testid="data-products-list">
+        <Col className="flex flex-wrap gap-2">
+          {!newLook && addTagButton}
```

Comment:
can check if it not returning zero and showing in UI when condition computed to false